app/assets/stylesheets/modules/_schedule.scss

```diff
@@ -383,3 +383,192 @@ input[type=number]::-webkit-outer-spin-button {
   background-color: white;
   z-index: 1000;
 }
+
```

Comment:
I like the nesting here. It doesn't look like it goes more than 4 deep, and only in a couple places. It's good to keep the nesting minimized (I usually shoot for max 3), as the autogenerated selectors can get unwieldy and complicate using dev tools.

Reply:
Thank you for this, I will definitely limit nesting in the future!